sway-lib-std/src/result.sw

```diff
@@ -25,6 +74,20 @@ impl<T, E> Result<T, E> {
     // Querying the contained values
     /////////////////////////////////////////////////////////////////////////
     /// Returns `true` if the result is [`Ok`].
+    /// 
+    /// # Examples
+    /// 
+    /// ```
+    /// enum Error {
+    ///     NotFound,
+    ///     Invalid,
+    /// }
+    /// 
+    /// let x: Result<u64, Error> = Result::Ok(42);
+    /// assert(x.is_ok());
+    /// 
+    /// let y: Result<u64, Error> = Result::Err(Error::NotFound));
+    /// assert(!x.is_ok());
```

Comment:
I think you'll need to close the code blobs with '```' too, I imagine `forc-doc` will complain one day.

Reply:
Corrected. Thank you